app/forms/keywords_form.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class KeywordsForm
+  include ActiveModel::Model
+
+  validates_with KeywordsFormValidator
+
+  attr_reader :file, :keyword_ids
```

Comment:
As we are not accessing these variables outside of this class, how about making them into private?

Reply:
We are accessing these variables in `KeywordsController` and `KeywordsFormValidator`, please help me to check it again

Reply:
Fixed in [`f84cf8d` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/f84cf8dbc89dd5a6e5ac9bb1503f04ae0923f7c4)